requirements.txt

```diff
@@ -14,3 +14,5 @@ click>=6.7
 psycopg2>=2
 python-Levenshtein>=0.12
 pathlib2>=2.3
+tabulate>=0.8.2
+pwgen
```

Comment:
Are these absolutely necessary? Every requirement we add weighs down all the modules.

Reply:
I guess...  and thing these are pretty lightweight / not heavy to install.

Reply:
I am 50:50 on this one. I think it is generally good if we exercise discipline on the `requirements.txt` for CatKit since the intention is mostly of this is mostly client code that hopefully runs on many different clusters/architectures. `pwgen` can be replaced by a simple function and I have just sent a PR to @kirstenwinther repository that does that. I think having some module like tabulate could be handy in several places. The module is pretty lightweight (single file, pure python). It supports several formats and seems quite versatile. If it can't be `tabulate` for some reason I would suggest `pandas`, which might be useful anyhow and is in weight maybe somewhere between `numpy` and `scipy` and `scipy` is already a dependency. Most forums I find on `pandas` in fact recommend `tabulate` for pretty-printing pandas DataFrames but `print(df)` already gives a somewhat readable output.